WultraMobileTokenSDK/Operations/Model/UserOperation/WMTOperationUIData.swift

```diff
@@ -45,13 +50,15 @@ open class WMTOperationUIData: Codable {
         blockApprovalOnCall = try? c.decode(Bool.self, forKey: .blockApprovalOnCall)
```

Comment:
We should probably replace reach `try?` with a try catch. because if the template is somehow not right on the backend, the developer has no idea why it didn't parse on the front end.

For example replace:
```swift
preApprovalScreen = try? c.decode(WMTPreApprovalScreen.self, forKey: .preApprovalScreen)
```

with

```swift
if c.contains(.preApprovalScreen) {
    do {
        preApprovalScreen = try c.decode(WMTPreApprovalScreen.self, forKey: .preApprovalScreen)
    } catch {
        D.error("Failed to decode \(Keys.preApprovalScreen) - \(error), setting to null")
        preApprovalScreen = nil
    }
} else {
    preApprovalScreen = nil
}
```